docs/pages/enroll-resources/application-access/cloud-apis/aws-console-without-agent.mdx

```diff
@@ -0,0 +1,304 @@
+
+This section will create the required AWS resources to enable the Profile Sync.
+
+1. Navigate to [Create Role](https://console.aws.amazon.com/iam/home?#/roles/create) and create a new IAM Role called `TeleportRolesAnywhereProfileSync`.
+   Trust policy must include the Roles Anywhere service principal:
+  ```json
+  {
+      "Version": "2012-10-17",
+      "Statement": [
+          {
+              "Effect": "Allow",
+              "Principal": {
+                  "Service": "rolesanywhere.amazonaws.com"
+              },
+              "Action": [
+                  "sts:AssumeRole",
+                  "sts:SetSourceIdentity",
+                  "sts:TagSession"
+              ]
+          }
+      ]
+  }
+  ```
+
+1. Copy the IAM Role ARN <Var name="Role ARN" />
+1. Navigate to the [TeleportRolesAnywhereProfileSync Role](https://console.aws.amazon.com/iam/home?#/roles/details/TeleportRolesAnywhereProfileSync) and create a new inline policy:
+  ```json
+  {
+      "Version": "2012-10-17",
+      "Statement": [
+          {
+              "Sid": "RolesAnywhereProfileSync",
+              "Effect": "Allow",
+              "Action": [
+            "rolesanywhere:ListProfiles",
+            "rolesanywhere:ListTagsForResource",
+            "rolesanywhere:ImportCrl",
+            "iam:GetRole"
+              ],
+              "Resource": [
+                  "*"
+              ]
+          }
+      ]
+  }
+  ```
+
+1. Now, navigate to the [Create a Profile](https://console.aws.amazon.com/rolesanywhere/home/profiles/create) page and name it `TeleportRolesAnywhereProfileSync`.
+Add the role created in the last step (`TeleportRolesAnywhereProfileSync`) and create the Profile.
+1. Copy the Profile ARN <Var name="Profile ARN" />
+
+### Create a Teleport AWS IAM Roles Anywhere integration
+Now that the required AWS resources are created, you can create a Teleport AWS IAM Roles Anywhere integration.
+
+1. Write the following contents to a file called `roles-anywhere-integration.yaml`:
+  ```yaml
+  kind: integration
+  sub_kind: aws-ra
+  version: v1
+  metadata:
+    name: <Var name="Integration Name" />
+  spec:
+    aws_ra:
+      trust_anchor_arn: <Var name="Trust Anchor ARN" />
+      profile_sync_config:
+        enabled: true
+        profile_arn: <Var name="Profile ARN" />
+        role_arn: <Var name="Role ARN" />
+  ```
+
+1. Create the integration with the following command:
+  ```code
+  $ tctl create -f roles-anywhere-integration.yaml
+  ```
+
+Teleport will now start syncing AWS IAM Roles Anywhere Profiles as AWS application resources, every 5 minutes.
+
+You can get the sync summary by running the following command:
+  ```code
+  $ tctl get integration/<Var name="Integration Name" />
+  ...
+  status:
+    aws_ra:
+      last_profile_sync:
+        end_time: "2025-07-23T15:46:27.475629Z"
+        error_message: ""
+        start_time: "2025-07-23T15:46:26.334536Z"
+        status: SUCCESS
+        synced_profiles: 4
+  ```
```

Comment:
I would move this to the bottom and have a "troubleshooting" section where I would include things like how to check integration status via web UI, and this CLI example as well.